src/main/java/org/tron/core/config/args/Args.java

```diff
@@ -330,6 +330,10 @@
   @Setter
   private String logLevel;
 
+  @Getter
+  @Setter
+  private int blockVersion;
+
   public static void clearParam() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this left curly brace to the beginning of next line of code. ([squid:LeftCurlyBraceStartLineCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6qTSgN4uCclo1ed&open=AWbJU6qTSgN4uCclo1ed)

[ISSUE_KEY]: AWbJU6qTSgN4uCclo1ed